pages/linux/ydotool.md

```diff
@@ -0,0 +1,16 @@
+# ydotool
+
+> Display server agnostic input automator.
+> More information: <https://github.com/ReimuNotMoe/ydotool>.
+
+- Left click with a mouse:
+
+`ydotool click 0xC0`
+
+- Right click with a mouse:
+
+`ydotool click 0xC1`
+
+- Send the keys Alt+F4:
```

Comment:
not too sure with this one, some feedback would be good.
```suggestion
- Input Alt and F4:
```

Reply:
The original is better because "Input Alt and F4" is more ambiguous on the release order. The key sequence here is "alt down", "f4 down", "f4 up", "alt up".

Reply:
I adjusted it a bit while keeping Alt+F4